src/sentry/quotas/base.py

```diff
@@ -70,40 +62,54 @@ def get_project_quota(self, project):
         max_quota_share = int(OrganizationOption.objects.get_value(
             org, 'sentry:project-rate-limit', 100))
 
-        if not legacy_quota and max_quota_share == 100:
-            return 0
-
-        org_quota = self.get_organization_quota(org)
+        if max_quota_share == 100:
+            return (0, 60)
```

Comment:
Yeah, that's cool. Didn't expect for this to get fixed now but probably worth looking at later.